osmatching/osmatching.py

```diff
@@ -166,7 +165,7 @@ def greedily_pick_matches(
     matches_per_case: int,
     matched_rows: pd.DataFrame,
     case_row: pd.DataFrame,
-    closest_match_variables: Union[None, list] = None,
+    closest_match_variables: list = [],
```

Comment:
It always make me a little bit nervous seeing mutable arg defaults, although I can see it's not an actual problem here. Would it work to use an empty tuple though?

Reply:
Actually, it doesn't need a default at all now that it's coming from the `MatchConfig` which already defaults to an empty list (using `field(default_factory=list)`), so I'll just remove it